py-scripts/lf_webpage.py

```diff
@@ -165,8 +294,8 @@ def phantom_check(self):
                                 # self.hostname_list.append(b['eid']+ " " +b['hostname'])
                                 devices_available.append(b['eid'] + " " + 'Win' + " " + b['hostname'])
                             elif "Linux" in b['hw version']:
-                                if ('ct' not in b['hostname']):
-                                    if ('lf' not in b['hostname']):
+                                if 'ct' not in b['hostname']:
```

Comment:
In the future, do not include readability changes in the same commits where there are logic changes. These should be separate to limit review burden.